src/cli/utils.ts

```diff
@@ -32,14 +37,16 @@ import * as spinner from './logIndicator';
 
 /** @hidden Current working directory reference */
 const WORKING_DIRECTORY = process.cwd();
+/** @hidden Config directory for running EOSIO */
+const CONFIG_DIRECTORY = path.join(__dirname, '../eosio-config');
 /** @hidden Temporary docker resource directory */
 const TEMP_DOCKER_DIRECTORY = path.join(__dirname, '.temp-docker');
 /** @hidden Slowest Expected test duration */
-const TEST_EXPECTED_DURATION = 2000;
+const TEST_EXPECTED_DURATION = 5000;
 /** @hidden Maximum test duration */
-const TEST_TIMEOUT_DURATION = 10000;
+const TEST_TIMEOUT_DURATION = 80000;
 /** @hidden Maximum number of EOS connection attempts before fail */
-const MAX_CONNECTION_ATTEMPTS = 8;
+const MAX_CONNECTION_ATTEMPTS = 20;
```

Comment:
How come all of these timeouts and attempts need to be so high? Are you hitting these numbers?

Reply:
I did have have some errors early using the tools which were hard to diagnose since they wouldn't happen on every run. Perhaps this could be set back to 8.